flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
It's the only thing I can test right now.

Reply:
clearly that won't change if you keep it that way.

Reply:
What do you suggest?

Reply:
obviously, to just remove it